<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomePage</class>
 <widget class="QWidget" name="WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>672</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>30</red>
        <green>29</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblIconLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblLogo">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	font: 36px &quot;HarmonyOS Sans&quot;;
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="text">
            <string>ZENO</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>19</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(255, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>by ZENUS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>52</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnNew">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>86</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>36</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>86</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string>NEW</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>24</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnOpen">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>86</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>36</height>
                </size>
               </property>
               <property name="text">
                <string>OPEN</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblRecentFiles">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Recent Files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblFileExample1">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>MyFlip</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblFileExample2">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Mountains</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblFileExample3">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>OpenGLStableFluid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMoreFile">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>more...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>55</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lblCurrVer">
               <property name="styleSheet">
                <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
               </property>
               <property name="text">
                <string>Current Version: 2.0.1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lblWhatsNew">
               <property name="styleSheet">
                <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
               </property>
               <property name="text">
                <string>Whatâ€™s New ?</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>43</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblLink">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Link:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblManual">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>ZENO Manual</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblOfficialWeb">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Zenus Official Web</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblForum">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Forum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblGithub">
             <property name="styleSheet">
              <string notr="true">QLabel
{
	font: 14px &quot;HarmonyOS Sans&quot;;
	color: rgb(133, 130, 128);
}</string>
             </property>
             <property name="text">
              <string>Project on GitHub</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
